module/VuFind/src/VuFind/Mailer/Mailer.php

```diff
@@ -180,83 +158,91 @@ public function setTransport($transport)
      *
      * @param string $input String to convert
      *
-     * @return AddressList
+     * @return array
      */
-    public function stringToAddressList($input)
+    public function stringToAddressList($input): array
     {
         // Create recipient list
-        $list = new AddressList();
+        $list = [];
         foreach (preg_split('/[\s,;]/', $input) as $current) {
             $current = trim($current);
             if (!empty($current)) {
-                $list->add($current);
+                $list[] = $current;
             }
         }
         return $list;
     }
 
     /**
-     * Constructs a {@see MimeMessage} body from given text and html content.
+     * Constructs a {@see Email} body from given text and html content.
```

Comment:
Should this be "an Email" rather than "a Email"?

Reply:
Fixed!